docs/source/notebooks/FAQ_dask_parallel.ipynb

```diff
@@ -19,7 +19,16 @@
    "metadata": {
```

Comment:
<p>"/home/tjs/mambaforge/envs/pavics-sdi/"  Careful, better refresh from Jenkins or directly on PAVICS to avoid personal path.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/pavics-sdi/pull/352/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>